camunda/v2/client.go

```diff
@@ -153,3 +154,57 @@ func (c *client) doPostRequest(ctx context.Context, params *bytes.Buffer, endpoi
 
 	return body, nil
 }
+
+func (c *client) DeleteTask(ctx context.Context, businessKey string) error {
+	tasks, err := c.getTasks(ctx, businessKey)
+	if err != nil {
+		return err
+	}
+	if len(tasks) == 1 {
+		if err = c.deleteProcessInstance(ctx, tasks[0].ProcessInstanceId); err != nil {
+			return err
+		}
+	} else {
+		return fmt.Errorf("found %d camunda tasks for businessKey: %s", len(tasks), businessKey)
+	}
+	return nil
+}
+
+func (c *client) getTasks(ctx context.Context, businessKey string) ([]Task, error) {
```

Comment:
(not relevant): this method looks like a good option to be public... maybe something like:
```suggestion
func (c *client) GetTasksByBusinessKey(ctx context.Context, businessKey string) ([]Task, error) {
```
wdyt?🤔